examples/Extensions/AddSitelinksUsingFeeds.cs

```diff
@@ -0,0 +1,366 @@
+            long? adGroupId = long.Parse("INSERT_AD_GROUP_ID_HERE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId, campaignId, adGroupId);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example adds sitelinks to a campaign using feed services. Run " +
+                       "AddCampaigns.cs to create a campaign.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The customer ID for which the call is made.</param>
+        /// <param name="campaignId">ID of the campaign to which sitelinks are added.</param>
+        /// <param name="adGroupId">Optional ID of the ad group to which sitelinks are added.</param>
+        public void Run(GoogleAdsClient client, long customerId, long campaignId, long? adGroupId)
+        {
+            try
+            {
+                // Create a feed, which acts as a table to store data.  
+                // feed_data = create_feed(client, customer_id)
+                Hashtable feedData = CreateFeed(client, customerId);
+
+                // Create feed items, which fill out the feed table with data.
+                CreateFeedItems(client, customerId, feedData);
+
+                // Create a feed mapping, which tells Google Ads how to interpret this data to
+                // display additional sitelink information on ads.
+                CreateFeedMapping(client, customerId, feedData);
+                
+                // Create a campaign feed, which tells Google Ads which campaigns to use the
+                // provided data with.
+                
+                CreateCampaignFeed(client, customerId, campaignId, feedData);
+                // If an ad group is specified, limit targeting only to the given ad group.
+                if (adGroupId.HasValue)
+                {
+                    CreateAdGroupTargeting(client, customerId, feedData, adGroupId.Value);
+                }
+            }
+            catch (GoogleAdsException e)
+            {
+                Console.WriteLine("Failure:");
+                Console.WriteLine($"Message: {e.Message}");
+                Console.WriteLine($"Failure: {e.Failure}");
+                Console.WriteLine($"Request ID: {e.RequestId}");
+                throw;
+            }
+        }
+
+        private Hashtable CreateFeed(GoogleAdsClient client, long customerId)
+        {
+            Feed feed = new Feed()
+            {
+                Name = $"Sitelinks Feed {DateTime.Now:yyyy-MM-dd hh:mm:ss}",
+                Origin = FeedOriginEnum.Types.FeedOrigin.User
+            };
+
+            // Specify the column name and data type. This is just raw data at this point,
```

Comment:
Could you check if the line length is wrapped to 100 chars?

Reply:
It is, yes.